### DISCUSSION TITLE
Masked Input

### DISCUSSION THREAD

**Discussion by juliolitwin** [BlazorComponent/MASA.Blazor#1749] [Q&A] [answered]
Hey,

In MudBlazor's TextField, there is an option for Masked Input, where I can add the mask. However, I couldn't find it in MASA. 
Is there really no support?

https://mudblazor.com/components/textfield#mask

**Comment by capdiem** [comment, marked as answer]
@juliolitwin Currently not supported. To track it better, I have created an issue for this feature.
